### PR TITLE
upd(git-credential-manager-core-deb): `2.0.696` -> `2.0.785`

### PR DESCRIPTION
**Pull request by indika-dev**
Git Credential Manager is outdated in pacstall-programs. This pull request updates the version number.

### CODE REVIEW

**Comment by Henryws** [comment]
Fix title.

File: packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name="git-credential-manager-core-deb"
 pkgname="git-credential-manager-core"
-gives="gcmcore"
+gives="gcm"
```
Comment (Henryws) [inline packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: Why are you changing this? Any specific reason?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript (hunk 1 above)
Comment (cat-master21) [reply packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: this is why https://github.blog/2022-04-07-git-credential-manager-authentication-for-everyone/ the name changed https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.785

Comment (indika-dev) [reply packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: Exactly what @cat-master21 mentioned. Additionally, they renamed the download artifact. That's why I changed gcmcore to gcm.

Comment (indika-dev) [reply packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: I'm sorry, but I don't understand, if I have to do some more things. @Henryws wrote 

> Requested changes must be addressed to merge this pull request.

As I understand this, this is meant for gcmcore <-> gcm. In my opinion this is already addressed?!

Comment (Henryws) [reply packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: The requested change being the title change.

Comment (Henryws) [reply packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:3]: @indika-dev.

**Comment by Henryws** [comment]
Still wrong title. It should be: upd(git-credential-manager-core-deb): `2.0.696` -> `2.0.785`.

File: packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 repology=("project: ${pkgname}")
-version="2.0.696"
+version="2.0.785"
 depends="git"
 url="https://github.com/GitCredentialManager/git-credential-manager/releases/download/v${version}/${gives}-linux_amd64.${version}.deb"
 description="Secure, cross-platform Git credential storage with authentication to GitHub, Azure Repos, and other popular Git hosting services"
-hash="caba73101f80c1a789225730d0d1b82941c31707cef6a55fb3cb3caada68d234"
+hash="8f79773116998fb2001a47b5a5266cbe1a737a54325bdab27d2ce6118a84ee7d
```
Comment (Henryws) [inline packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript:9]:
```suggestion
hash="8f79773116998fb2001a47b5a5266cbe1a737a54325bdab27d2ce6118a84ee7d"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**